src/gwt/src/org/rstudio/studio/client/rsconnect/ui/RSConnectDeploy.java

```diff
@@ -214,23 +221,7 @@ public RSConnectDeploy(RSConnectPublishSource source,
                      }
                   });
          });
-      }
```

Comment:
unreachable and unnecessary clause based on prior if clause -- fixed from a previous PR, but never removed the dead code